Dispatcher/Supporting Files/Assets.xcassets/settings.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Vector (1).png",
```

Comment:
We better not upload images with file names that contain stuff like (1) or spaces.
Should rename everything to something more meaningful.